packages/table/src/TableHeader.tsx

```diff
@@ -29,7 +29,6 @@ export const TableHeader = ({
 			color="text"
 			padding={0.75}
 			fontWeight="bold"
-			borderBottom
```

Comment:
We don't need a `borderBottom` here because a border is set on the `TableHead` component